pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -121,6 +132,7 @@ func doCreateBridge(bridgeName string, bridgeNum int,
 		return errors.New(errStr), ""
 	}
 	disableIcmpRedirects(bridgeName)
+	enableVlanFiltering(bridgeName)
```

Comment:
There is a question in my mind whether we also want to do this for the ethN bridges created by nim (and set up kethN and all the ports attached to those bridges to allow all vlans by default.)
Would that make it be more uniform?

Reply:
@gkodali-zededa thoughts on the above?

Reply:
Ports that attach to ethN bridges are either kethN port or app ports connected to switch network instances. Taking care of the vlans of vif (switch NI) is already take care.
So, I guess the remaining portion is to attach vlan range 1-4093 to kethN ports in NIM. Do you agree?

Reply:
I did not have to set the vlan range on keth0